lib/computer_player.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "player"
+
+class ComputerPlayer < Player
+
+  attr_reader :player_input
+
+  def get_input
+    @display.present(Messages.new.player_prompt(@name))
+    randomise
+    display_input
+  end
+
+  def randomise
+    options = ["rock", "paper", "scissors"]
+    @player_input = options.sample
+  end
+
+  def display_input
```

Comment:
Is that really display an input correct in this case? as a value _inserted_
Wouldn't something like display_shape more accurate?